connector/connect/server/src/main/scala/org/apache/spark/sql/connect/planner/SparkConnectPlanner.scala

```diff
@@ -2852,6 +2877,7 @@ class SparkConnectPlanner(val sessionHolder: SessionHolder) extends Logging {
         throw new IllegalArgumentException("Missing command in StreamingQueryCommand")
     }
 
+    executeHolder.events.postFinished()
```

Comment:
@rangadi

Reply:
Again, it is not clear to me why we want to do this for each handler separately. But if you think this is the right way to do it, go head.